### ISSUE TITLE
Why did you decide to move to Medium?

### ISSUE THREAD

**Issue by shairez** [kentcdodds/ama#357]
Hey buddy 😄 

And a followup - how is the experience so far?

**Comment by kentcdodds** [comment]
Hey @shairez,
Thanks for the question.

I decided to move to Medium because unless I want to learn something specific, I prefer to buy rather than build. I'm in the business of learning, building, and teaching. I'm not interested in learning how to build a blogging platform. I'm just trying to get content out there in a way that reaches people. So rather than spend weeks building a blog and years maintaining it, I'd rather just use Medium.

On top of that, Medium is a good way to get a lot of eyes on your stuff. I don't have that need quite as much anymore because I have a pretty good following at this point. But if you're considering things for yourself, just know that you'll probably find more people reading your articles on Medium than your own blog.

Another important note. I signed up for Medium when it was still possible to get Medium with a custom domain name. I've heard that's not possible anymore. If I hadn't been able to do that then I wouldn't have gone with Medium. It's very important to me that I control the domain where my blog content lives. This way if I decide to move off of Medium, I can easily make a bunch of redirects from the blog posts where they used to live to where they now live on the new platform.

I hope that's helpful!
